transforms/class.js

```diff
@@ -12,6 +12,57 @@
 
 module.exports = (file, api, options) => {
   const j = api.jscodeshift;
+  const util = require('util');
+
+  // Coursera specific sort order, taken from our eslintrc
+  const COMPONENT_METHOD_ORDER = [
```

Comment:
yeah that'd be good.

i'd love to be able to hook into nuclide-format-js or similar things for other editors to automatically sort components on save